NEW_README.md

```diff
@@ -1,132 +1,308 @@
-Apply this configuration with the following command:
-```
-source config/env/env_setup_2019.0.1
-```
 
-#### Provision, tune, run
+
+### Provision, tune, run
+
 A set of 'autoscale' rake tasks are provided to handle setup and scale test execution.
 The pre-suite has been updated to include tuning of the master via 'puppet infrastructure tune' for scale tests so this is no longer a manual step.
-As with the other configurations test nodes can be provisioned as part of the run or as a separate step.
+As with the other configurations tests,  nodes can be provisioned as part of the run or as a separate step.
 
 Note that when using the autoscale rake tasks the `BEAKER_PRESERVE_HOSTS` environment variable is always set to 'true', so you will need to de-provision the test nodes with the `performance_deprovision_with_abs` when your testing is complete.
 
-* To provision nodes as part of the run:
+To provision nodes as part of the run:
 ```
 bundle exec rake autoscale
 ```
 
-* To provision nodes separately:
+#### To provision nodes separately:
 
-run the `autoscale_setup` rake task to provision the nodes:
+Run the `autoscale_setup` rake task to provision the nodes:
 ```
 bundle exec rake autoscale_setup
 ```
 
-then run the 'autoscale_provisioned' rake task to run the scale test:
+Then run the 'autoscale_provisioned' rake task to run the scale test:
 ```
 bundle exec rake autoscale_provisioned
 ```
 
-* De-provision the nodes when testing is complete:
+#### De-provision the nodes when testing is complete
+
 ```
 bundle exec rake performance_deprovision_with_abs
 ```
 
+#### Smaller autoscale tasks
+
 There are additional rake tasks for small and medium autoscale runs to allow testing of the environment and autoscale functionality without waiting for a full run:
-* autoscale_provisioned_tiny
+
+`autoscale_provisioned_tiny`
 - 1 agents
 - 3 iterations
 - increment by 1
 
-* autoscale_provisioned_sm
+`autoscale_provisioned_sm`
 - 10 agents
 - 10 iterations
 - increment by 10
 
-* autoscale_provisioned_med
+`autoscale_provisioned_med`
 - 500 agents
 - 10 iterations
 - increment by 100
 
-### Acceptance tests
-You can execute the 'acceptance' rake task which will run everything in VMPooler rather than AWS and do a much shorter gatling run. This is useful for quickly testing changes to the performance test setup. If you need to execute acceptance tests in the AWS environment, you can set the following env vars:
 
-`ABS_OS=centos-7-x86-64-west BEAKER_HOSTS=config/beaker_hosts/pe-perf-test.cfg`
+## Acceptance tests
+
+You can execute the `acceptance` rake task resulting in a much shorter gatling
+run.  This task runs VMPooler by default.  This is useful for quickly testing
+changes to the performance test setup.
+
+The [expected environment variables](#environment-variables) must be set prior
+to executing this task.  You must also set an an environment variable for
```

Comment:
setting BEAKER_OPTIONS_FILE isn't a "must" as the rake file sets if for you based on the install type...